src/App/Application.cpp

```diff
@@ -1975,264 +1951,264 @@ void Application::init(int argc, char ** argv)
 void Application::initTypes()
 {
     // Base types
-    Base::Type                      ::init();
-    Base::BaseClass                 ::init();
-    Base::Exception                 ::init();
-    Base::AbortException            ::init();
-    Base::Persistence               ::init();
+    Type                      ::init();
+    BaseClass                 ::init();
+    Exception                 ::init();
+    AbortException            ::init();
+    Persistence               ::init();
 
     // Complex data classes
     Data::ComplexGeoData            ::init();
     Data::Segment                   ::init();
 
     // Properties
     // Note: the order matters
-    App::Property                   ::init();
-    App::PropertyContainer          ::init();
-    App::PropertyLists              ::init();

```

Comment:
These doesn't align anymore.

We should either align them or remove the whitespaces between class name and `::init()`.
I would align them to avoid discussions about changing alignment conventions in this PR.

Reply:
My understanding is that Werner wants to manually check all changes done with the tool. I might be wrong, but I'm quite sure he prefers it to be aligned. Either way, it's best to avoid breaking alignment for now.